### PR TITLE
Docs for tagging tests

### PR DESCRIPTION
**Pull request by Luc45**
Part of CR

### CODE REVIEW

File: docs/custom-tests/02-tagging-tests.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+:::info
+The Custom Tests feature is coming soon.
+:::
+
+## Introduction
+
+You can use test tags to run specific tests from a plugin.
+
+Each plugin can have one or multiple test tags, and we also offer some generic tests that can be used by any extensions.
```
Comment (zhongruige) [inline docs/custom-tests/02-tagging-tests.md]: Minor nitpick:

```suggestion
Each plugin can have one or multiple test tags, and we also offer some generic tests that can be used by any extension.
```

We also use extensions and plugins both on this page, should we maybe use `plugin` here instead? What do you think?

Comment (Luc45) [reply docs/custom-tests/02-tagging-tests.md]: Hmm, I'm not sure.... I don't like "extension" very much, but themes are theoretically supported, you just have to run them with `--testing_theme` (I plan to get rid of this requirement at a later point).

Comment (Luc45) [reply docs/custom-tests/02-tagging-tests.md]: That flag is required when the SUT is a theme

Comment (Luc45) [reply docs/custom-tests/02-tagging-tests.md]: Just pushed a commit, what do you think?

Comment (zhongruige) [reply docs/custom-tests/02-tagging-tests.md]: I'm in the same boat, I tend to prefer plugin over extension generally myself.

Comment (Luc45) [reply docs/custom-tests/02-tagging-tests.md]: I'll add a dedicated page about themes with all the nitpicks

Comment (zhongruige) [reply docs/custom-tests/02-tagging-tests.md]: > Just pushed a commit, what do you think?

Those tweaks look good to me!

Comment (Luc45) [reply docs/custom-tests/02-tagging-tests.md]: I've added a page about Themes as well

Comment (Luc45) [reply docs/custom-tests/02-tagging-tests.md]: > Those tweaks look good to me!

All right, so I'm merging away

**Pull request merged**